docs/installation.rst

```diff
@@ -71,11 +71,11 @@ maintain the order due to database relational constraints:
     'mptt',
     'sekizai',
     'sorl.thumbnail',
-    'wiki',
-    'wiki.plugins.attachments',
-    'wiki.plugins.notifications',
-    'wiki.plugins.images',
-    'wiki.plugins.macros',
+    'wiki.apps.WikiConfig',
+    'wiki.plugins.attachments.apps.AttachmentsConfig',
+    'wiki.plugins.notifications.apps.NotificationsConfig',
+    'wiki.plugins.images.apps.ImagesConfig',
+    'wiki.plugins.macros.apps.MacrosConfig',
```

Comment:
Is there some new consensus in Django documentation that `INSTALLED_APPS` should specify application configuration names for external apps with only 1 default configuration?

Reply:
I don't know, but I relies on docs to avoid `default_app_config` https://docs.djangoproject.com/en/2.0/ref/applications/#configuring-applications

`New applications should avoid default_app_config. Instead they should require the dotted path to the appropriate AppConfig subclass to be configured explicitly in INSTALLED_APPS.`